packages/engine/crawlers/solana/index.ts

```diff
@@ -19,6 +20,15 @@ export const solanaEngineRunner: SolanaRunner = {
 				currentPubkey,
 			);
 
+			const { _id } = item;
+			getSignatureList(context.connection, _id)
+				.then(async (list) => {
+					await getTransactions(context, list, _id);
+				})
+				.catch((error) => {
+					console.log(error);
+				});
```

Comment:
then, catch should be within getSignatureList itself -- if it make `getSignatureList` become too ambitous, split that function into smaller functions, but from within this `solana/index.ts` level, all we need is to call something very straight forward and does all the job.